mne/viz/evoked.py

```diff
@@ -1482,6 +1499,10 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
         determines the CI width. E.g., if this value is .95 (the default),
         the 95% parametric confidence interval is drawn.
         If None, no shaded confidence band is plotted.
+    ci_alpha : float
+        The alpha value (opacity) of the confidence band.
+    ci_method : str
+        Compute parametric (scipy.t.interval) or Bayesian (scipy.stats.bayes_mvs) confidence band. Should be "parametric" or "bayesian".
```

Comment:
This will be problematic for PEP8 (line length). Also use
```
:func:`scipy.t.interval`
:func:`scipy.stats.bayes_mvs`
```
It will link to their doc properly.